packages/flutter/lib/src/services/autofill.dart

```diff
@@ -13,11 +13,7 @@ export 'text_input.dart' show TextEditingValue, TextInputClient, TextInputConfig
 /// Each hint is pre-defined on at least one supported platform. See their
 /// documentation for their availability on each platform, and the platform
 /// values each autofill hint corresponds to.
-class AutofillHints {
-  // This class is not meant to be instantiated or extended; this constructor
-  // prevents instantiation and extension.
-  AutofillHints._();
-
+abstract final class AutofillHints {
```

Comment:
Most of these private constructors were removed once the `final` class modifier was added; I finished the rest in this PR.